myEnigma/myEnigma.ino

```diff
@@ -724,7 +724,7 @@ HT16K33 HT;
 #define DEFVALB   0x07 // 
 #define INTCONA   0x08 // Interrupt on change control register
 #define INTCONB   0x09 // 
-#define IOCON     0x0A // Control register
+//#define IOCON   0x0A // Control register
 #define IOCON     0x0B // 
```

Comment:
Reading the doc I see that it is no A/B because it is the same register (page 10 in http://ww1.microchip.com/downloads/en/DeviceDoc/21952b.pdf) so having it defined twice is overkill/wrong. 
To better understand that when looking at the code in a year or so I like the first one kept and the second one commented out with a (second)  comment that it's a dup.